integration/duckdb/README.md

```diff
@@ -0,0 +1,51 @@
+# Lance DuckDB Extension
+
+*Linux Only*
+
+## Functions
+
+List / Array functions
+
+| Function            | Description                        |
+|---------------------|------------------------------------|
+| `list_argmax(list)` | Run `argmax` over a list of values |
```

Comment:
add this is only for floats? Let's also file a duckdb github issue. This should just be a native duckdb function

Reply:
It supports `int/int64/float/double`.

Sure can file an issue upstream.